packages/react-heartwood-components/src/components/Card/components/CardHeader.tsx

```diff
@@ -20,8 +20,15 @@ export interface ICardHeaderProps
 	contextMenu?: IContextMenuProps | null
 }
 
-const CardHeader = (props: ICardHeaderProps): React.ReactElement => {
-	const { title, labelText, labelIcon, actions, contextMenu } = props
+const CardHeader = (
+	props: ICardHeaderProps | IHWCardHeader
+): React.ReactElement => {
+	const propsComp = props as ICardHeaderProps
+	const propsGql = props as IHWCardHeader
+
+	const { title, labelText, actions, contextMenu } = propsGql
+	const { labelIcon } = propsComp
```

Comment:
Per convo, everything you've changed as a part of `ICardHeaderProps` should go in here; if there's a substantial divergence in the two types we can do `foo as commonFoo` and `foo as reactHeartwoodFoo` etc. to craft the logic, which would again preserve DX comprehension w/o a ton of mental leaps :D